build-pyspy.bat

```diff
@@ -0,0 +1,56 @@
+@echo off
```

Comment:
Let's place the Windows scripts under `scripts/windows/`

Reply:
Done

Reply:
Not pushed?

Reply:
It should be pushed now

Reply:
Scripts are still under `scripts/` and not `scripts/windows/` now
![image](https://user-images.githubusercontent.com/8831572/200092432-cc5a0984-ee9f-4160-bd11-24d997c16113.png)